content/docs/command-reference/version.md

```diff
@@ -55,17 +53,15 @@ The detail of DVC version depends upon the way of installing DVC.
   to publish yet. Therefore, installing with this method might result in usage
   issues. To trace any error reported with this setup, we need to know exactly
   which version is being used. For this we rely on a Git commit hash that is
-  displayed in the output of `dvc version` like this: `0.40.2+292cab.mod`. The
+  displayed in the output of `dvc version` like this: `1.7.4+292cab.mod`. The
   part before `+` is the `_BASE_VERSION`, and the following part is the commit
   hash of the tip of the `master` branch. The optional suffix `.mod` means that
   code is modified.
 
-### What we mean by "Binary"
+- **Package manager or Binary**: This part shows how DVC was downloaded and
+  [installed](/doc/install).
```

Comment:
This sentence is correct, thanks. But notice we don't often complete sentences starting with use upper case after `:` for some reason. Let's keep that style for consistency (just commit this):

```suggestion
- **Package manager or Binary**: shows how DVC was downloaded and
  [installed](/doc/install).
```

Reply:
^ Also notice corrected case of "binary".

Reply:
Should "Binary" be mentioned before "package manager" though? Or the list below reordered (it starts with binary types). I think the latter.

Reply:
> it starts with binary types

True "Binary" should actually come before "package manager".